FabLabKasse/faucardPayment/MagPosLog.py

```diff
@@ -34,13 +34,13 @@ def __init__(self, amount, cur, con):
         self.payed = False
         self.cur = cur
         self.con = con
-        self.amount = float(amount)
+        self.amount = amount
         self.timestamp_payed = None
         self.oldbalance = 0
         self.newbalance = 0
 
         # Create table if it does not exist
-        self.cur.execute("CREATE TABLE IF NOT EXISTS MagPosLog(id INTEGER PRIMARY KEY AUTOINCREMENT, datum, cardnumber INT, amount FLOAT, oldbalance INT, newbalance INT, timestamp_payed, status INT, info INT, payed)")
+        self.cur.execute("CREATE TABLE IF NOT EXISTS MagPosLog(id INTEGER PRIMARY KEY AUTOINCREMENT, datum, cardnumber INT, amount TEXT, oldbalance INT, newbalance INT, timestamp_payed, status INT, info INT, payed)")
```

Comment:
SQLite cannot (?) change the value of a column, and conversion will be quite annoying. I think we should rather accept storing the values as float and load them with float_to_decimal. As long as you don't go above 1e8 euros or so, there is no problem with float as long as you round correctly.

If you really do want to convert the database, then we could just switch to INT and store it as cents.